### PR TITLE
feat(LT-4958): add StatusChanged action type.

### PR DESCRIPTION
**Pull request by fatihdumanli**
Add 'StatusChanged' action type as per the requirements of the CR LT-3921.

### CODE REVIEW

File: src/Lykke.Snow.Audit/AuditEventType.cs [hunk 1]
```diff
@@ -7,6 +7,7 @@ public enum AuditEventType
     {
         Edition,
         Creation,
-        Deletion
+        Deletion,
+        StatusChanged
```
Comment (tarurar) [inline src/Lykke.Snow.Audit/AuditEventType.cs:11]: Well, this is ambiguous. You already have event type `Edition` and this enum is supposed to have high level event type only. Having `Edition` and `StatusChanged` simultaneously is confusing.

Comment (fatihdumanli) [reply src/Lykke.Snow.Audit/AuditEventType.cs:11]: This is team's common decision, we agreed that if the business has such a requirement, we should have such an enum type.

Please feel free to schedule a call to review again if you think it is necessary.

Comment (tarurar) [reply src/Lykke.Snow.Audit/AuditEventType.cs:11]: Business doesn't express its requirements in technical terms as the enum is. This particular enum is an implementation details and implementation details inconsistencies can't be excused by business requirements.

Comment (anokhin-lkk) [reply src/Lykke.Snow.Audit/AuditEventType.cs:11]: @tarurar this is not ambiguous. "Edition" and "StatusChange" have different business meaning

**Review by anokhin-lkk — APPROVED** [review]

File: src/Lykke.Snow.Audit/AuditEventType.cs (hunk 1 above)
Comment (anokhin-lkk) [reply src/Lykke.Snow.Audit/AuditEventType.cs:11]: @tarurar if you are working, let's have a meeting and discuss that

**Pull request merged**